CMakeLists.txt

```diff
@@ -54,26 +57,36 @@ if (GLOW_WITH_OPENCL)
   find_package(OpenCL REQUIRED)
 endif ()
 
-# Prefer LLVM 6.
-find_package(LLVM 6 CONFIG)
-# If LLVM 6 not found, try build with whatever is available.
-if (NOT LLVM_FOUND)
-  find_package(LLVM CONFIG)
-endif()
-if (NOT LLVM_FOUND)
-  message(FATAL_ERROR "Could not find LLVM. Build LLVM manually and configure the project with -DCMAKE_PREFIX_PATH=/path/to/llvm/install")
+if(MSVC)
+  set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} /wd4710 /wd4711 /wd4514 /wd4571 /wd4820 /wd4774 /wd4625")
+  set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} /wd4526 /wd4061 /wd4365 /wd4242 /wd4996 /wd5026 /wd4626")
+  set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} /wd5027 /wd4668 /wd4305 /wd4826 /wd4574")
+else()
+  # Prefer LLVM 6.
+  find_package(LLVM 6 CONFIG)
```

Comment:
Something doesn't make sense here... llvm is a required dependency of glow.  I don't understand how you can possibly be building without it.

Reply:
@bertmaher , Win32 have no llvm-devel package, I building glow with llvm source.
[